ocaml/xapi-aux/kerberos_encryption_types.ml

```diff
@@ -28,6 +46,20 @@ module Winbind = struct
     | All ->
         "all"
 
+  let add x y = x lor y
```

Comment:
You might get nicer-looking code by using `let (+++) = x lor y`, it's a shame that (add) can be used as an infix.

Reply:
No, `add` cannot be used as infix. 
but I can take your advice to use `+++` as infix instead. Thanks.